WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -8,16 +8,18 @@ struct NavigationItemRow : ImmuTableRow {
     let title: String
     let icon: UIImage?
     let action: ImmuTableAction?
+    let showsDisclosureIndicator: Bool
```

Comment:
I like that idea! I've changed it to take an accessory type, but default to `.DisclosureIndicator`.